### ISSUE TITLE
write up the story of take-what-you-want compensation

### ISSUE THREAD

**Issue by whit537** [gratipay/inside.gratipay.com#552]
Take-what-you-want compensation is the most interesting thing we've discovered. We should write up a case study of our first go-round, to help spread the word about the new version.

See https://github.com/gratipay/inside.gratipay.com/issues/534#issuecomment-201423158 and https://github.com/gratipay/inside.gratipay.com/issues/551 for places this would/could land.

**Comment by whit537** [comment]
A brief history of Gratipay teams (https://github.com/rubygems/rubygems.org/issues/500#issuecomment-65147249):

> I just did some spelunking: Gratipay's [teams](http://gratipay.com/about/teams) feature began life in [this commit from April 11, 2013](https://github.com/gratipay/gratipay.com/commit/83c7eaa73a594f72946be868fb90fdcb0e0bd3eb#diff-1f540dca5c867f9928789a5c48fd6d98), reached its first point of stability [a few days later](https://github.com/gratipay/gratipay.com/commit/2c509a8d23a074aeacbdf00c7dee89cc32024677), got a rewrite a couple weeks later, [on May 1](https://github.com/gratipay/gratipay.com/issues/901), and then a second rewrite to its present form two months after that, [on July 1](https://github.com/gratipay/gratipay.com/issues/1100) (with a blog post [two days later](https://medium.com/gratipay-blog/team-gittip-eba0a27825b8)). This present ticket is from December 13, 2012, four months before any sort of teams feature on Gratipay. The conversation started even earlier, [on August 4, 2012](https://github.com/gratipay/gratipay.com/issues/223), seven months before teams.

**Comment by whit537** [comment]
Alright, so it looks like we ran with twyw for almost *two* years (I had thought it was only one): from July 3, 2013 through the [Gratipocalypse](https://gratipay.news/gratipocalypse-42fd0ec0d9e8), May 7, 2015.

**Comment by whit537** [comment]
How much money did we move? How many teams participated? It'd be good to look at other teams besides Gratipay, though Gratipay went the furthest with it.

**Comment by whit537** [comment]
We have all the data in the `transfers` table under `context = 'take'`. 17,642 rows between July 4, 2013, and April 30, 2015.

**Comment by whit537** [comment]
- All told, 117 teams distributed $49,339.20 to 336 individuals.
 - Gratipay distributed $21,656.61 to 102 members over 96 weeks.
 - DrupalCoreTeam distributed $13,939.35 to 22 members over 52 weeks (starting May 8, 2014).
 - All the rest had less then 10 members each.

| volume ($) | teams |
|---------:|----------:|
| 10,000.00 - 21,656.61 | 2 |
| 1,000.00 - 9,999.99 | 4 |
| 100.00 - 999.99 | 19 |
| 0.00 - 99.99 | 92 |

| volume ($) | members |
|---------:|----------:|
| 1,000.00 - 6,412.10 | 13 |
| 100.00 - 999.99 | 46 |
| 0.00 - 99.99 | 276 |

**Comment by whit537** [comment]
**tl;dr**: 300+ people took about $50,000 from 100+ teams over almost two years.

(59 people and 25 teams account for most of that.)

**Comment by justinxreese** [comment]
cc @swans-one

**Comment by whit537** [comment]
The strongest part of the story is Gratipay's own experience with it. We had 100+ people split over $20,000 over the course of almost two years. Maybe we should focus on those numbers instead of "300 / 50,000 / 100"?

**Comment by whit537** [comment]
> The strongest part of the story is Gratipay's own experience with it.

It's also the one we have the most experience with. ;-)

For Drupal's part of the story, we can refer to https://www.youtube.com/watch?v=ttWfSC2m7Ic.

**Comment by whit537** [comment]
I rewatched the [video w/ Drupal Core ](https://www.youtube.com/watch?v=ttWfSC2m7Ic) from 18 months ago, and made [some rough notes](https://medium.com/@Gratipay/rough-notes-from-https-www-youtube-com-watch-v-ttwfsc2m7ic-a693d8292efe). They're an important part of the take-what-you-want story.

**Comment by whit537** [comment]
Rewrite: "[Take What You Want](https://medium.com/@Gratipay/b62a40df4360)." Dropped the airplane motif. More concrete details about how payroll actually worked.

**Comment by whit537** [comment]
I reached out to @YesCT via private email (hi if you're watching GitHub notifications! :-) for a link to the criteria they used for the old Drupal Gittip team.

**Comment by whit537** [comment]
Mail bounced.

**Comment by whit537** [comment]
How may customers have we had?

"We’ve processed over $1,000,000 in payments on behalf of N customers."

We have 176 teams currently, but of course most of the $1,000,000 was under Gittipay 1.0. How about ...

```sql
=> select count(*) from (select distinct on (participant) participant from exchanges where amount < 0) as _;
┌───────┐
│ count │
├───────┤
│  1592 │
└───────┘
(1 row)
```

**Comment by whit537** [comment]
Part of the importance of writing up this story is that it frees us up to let go of the `takes` table in the database, which looks like it will happen in https://github.com/gratipay/gratipay.com/pull/4018.

**Comment by whit537** [comment]
#683 will close this.
